packages/suite/src/components/settings/SettingsMenu/index.tsx

```diff
@@ -1,14 +1,14 @@
 import React, { useState } from 'react';
 import styled from 'styled-components';
 import { Dropdown } from '@trezor/components';
-import { Translation, ExternalLink, AppNavigationPanel, AppNavigation } from '@suite-components';
+import { Translation, TrezorLink, AppNavigationPanel, AppNavigation } from '@suite-components';
 import { useActions, useSelector } from '@suite-hooks';
 import * as modalActions from '@suite-actions/modalActions';
 import * as suiteActions from '@suite-actions/suiteActions';
 import * as routerActions from '@suite-actions/routerActions';
 import { SUPPORT_URL } from '@suite-constants/urls';
 
-const StyledLink = styled(ExternalLink)`
+const StyledLink = styled(TrezorLink)`
```

Comment:
It should not be underlined.
![Screenshot 2021-09-07 at 14 27 58](https://user-images.githubusercontent.com/33235762/132344520-69d4a4b3-3722-49fc-abcd-865d5241c53d.png)
![Screenshot 2021-09-07 at 14 28 04](https://user-images.githubusercontent.com/33235762/132344532-e750da4f-2a46-43c3-94f3-39e5bbf69638.png)

Reply:
❌ @ravinalamada Did you test it?